config/routes.rb

```diff
@@ -12,25 +12,28 @@
 #    resources :registered_users, only: [:update]
   end
 
- # namespace :registered_users, path: ":username" do
- #  #  resources :boards, only: [:index, :new, :create]
- #   get '/:name', to: 'boards#show', as: 'board_path'
- # #   get '/:name/edit', to: 'boards#edit'
- # #   patch '/:name', to: 'boards#update'
- # #   delete '/:name', to: 'boards#destroy'
- # #   post '/:name/like', to: 'like#create'
- # end
+#  namespace :registered_users, path: ":username" do
+#    resources :boards, only: [:index, :new, :create]
+#    get '/:name', to: 'boards#show'
+#    get '/:name/edit', to: 'boards#edit'
+#    patch '/:name', to: 'boards#update'
+#    delete '/:name', to: 'boards#destroy'
+#    post '/:name/like', to: 'like#create'
+#  end
+
+
+  get '/o/oauth2/auth', as: :google_login
```

Comment:
If we're only using this route for testing, can we name is as google login for testing? Will it let us do that?

It would be great if we could add the real login path and call it google_login_path and refer to it with the link helper on the sign in page:
get '/auth/google_oauth2', as: :google_login
get '/o/oauth2/auth', as: :google_test_login